pymc/variational/approximations.py

```diff
@@ -52,20 +53,20 @@ class MeanFieldGroup(Group):
     short_name = "mean_field"
     alias_names = frozenset(["mf"])
 
-    @node_property
+    @cached_property
```

Comment:
these are all just dictionary lookups (or construction of symbolic variables). I don't think it's necessary to cache them